services/noop/flags.go

```diff
@@ -18,14 +18,23 @@
 package noop
 
 import (
+	"encoding/json"
+
 	"github.com/mysteriumnetwork/node/core/service"
 	"github.com/urfave/cli"
 )
 
 // Options describes options which are required to start Noop service
 type Options struct{}
 
-// ParseFlags function fills in Noop options from CLI context
-func ParseFlags(_ *cli.Context) service.Options {
+// ParseCLIFlags function fills in Noop options from CLI context
+func ParseCLIFlags(_ *cli.Context) service.Options {
 	return Options{}
 }
+
+// ParseJSONOptions function fills in Noop options from JSON request
+func ParseJSONOptions(request json.RawMessage) (service.Options, error) {
```

Comment:
Renamed it to `options.go` since we have options filling there, both for arguments and JSON requests. It's about service options only, not the whole service request.